### ISSUE TITLE
Eval returns length of an empty string as undefined

### ISSUE THREAD

**Issue by baruinho** [TechnologyAdvice/Jexl#17]
```
var jexl = require("jexl");
jexl.eval("name.length", {name: ""}).then(console.log);
// undefined
jexl.eval("name.length == 0", {name: ""}).then(console.log);
// false
jexl.eval("name.length == undefined", {name: ""}).then(console.log);
// true
```

**Comment by TomFrost** [comment]
Confirmed! Will dive in.

**Comment by TomFrost** [comment]
Fix written, release pending.

**Comment by TomFrost** [comment]
Published fix in 1.1.4; will be rolled in to the 2.0 branch soon.

**Comment by baruinho** [comment]
That's great! Thank you
